docs/ref/contrib/gis/gdal.txt

```diff
@@ -1413,6 +1413,36 @@ blue.
 
         The total number of pixels in this band. Is equal to ``width * height``.
 
+    .. method:: statistics(refresh=False, approximate=False)
+
+        .. versionadded:: 1.10
+
+        Compute statistics on the pixel values of this band. The return value
+        is a tuple with the following structure:
+        ``(minimum, maximum, mean, standard deviation)``.
+
+        If the ``approximate`` argument is set to ``True``, the statistics may
+        be computed based on overviews or a subset of image tiles.
+
+        If the ``refresh`` argument is set to ``True``, the statistics will be
+        computed from the data directly, and the cache will be updated with the
+        result.
+
+        Note that if a persistent cache value is found, that value is returned.
+        For raster formats using Persistent Auxiliary Metadata (PAM) services,
+        the statistics might be cached in an auxiliary file. In some cases this
+        metadata might be out of sync with the pixel values or store
```

Comment:
maybe "or cause values from a previous call to be returned which don't reflect the value of the approximate argument" (assuming this problem can happen both ways)?